run.go

```diff
@@ -178,3 +171,27 @@ type runMountArtifacts struct {
 	// LockedTargets to be unlocked if there are any.
 	LockedTargets []string
 }
+
+// RunMountSources are the availible run mounts for this run
+type RunMountSources struct {
```

Comment:
Nit, not-blocker: I think as of now it holds  attributes other than sources as well, could we call it some thing like `RunMountInfo` or `RunMountAttributes` so it becomes more generic. But lets wait for others to confirm.

Reply:
I'd lean towards RunMountInfo fwiw.